misc/DetectOtherLanguages.ipynb

```diff
@@ -0,0 +1,188 @@
+{
```

Comment:
<p>I feel like that this notebook is no longer needed? Or we should move all the Twitter data processing to a separate file/folder?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>So that would be DetectOtherLanguages and TweetMining right?</p>